scripts/bench-single-owner-multikey.py

```
@@ -0,0 +1,235 @@
+    # "buf += chunk; buf.count(...)" is O(n^2) and dominated both the time and
+    # the variance of the first version of this harness.
+    #   SET reply            "+OK\r\n"                      =  5
+    #   MGET reply (2 x "v") "*2\r\n$1\r\nv\r\n$1\r\nv\r\n"      = 18
+    per_batch = 5 + 5 + 18
+    need = batches * per_batch
+    t0 = time.perf_counter()
+    s.sendall(payload)
+    got, saw_err = 0, False
+    while got < need:
+        chunk = s.recv(1 << 20)
+        if not chunk: raise SystemExit("connection closed mid-drain")
```

Ruff (0.16.2)

[error] Multiple statements on one line (colon)

(E701)